pkg/config/config.go

```diff
@@ -7,11 +7,15 @@ import (
 
 // Controller defines the standard config for this operator.
 type Serialized struct {
-	Report      bool   `json:"report"`
-	StoragePath string `json:"storagePath"`
-	Interval    string `json:"interval"`
-	Endpoint    string `json:"endpoint"`
-	Impersonate string `json:"impersonate"`
+	Report           bool   `json:"report"`
+	StoragePath      string `json:"storagePath"`
+	Interval         string `json:"interval"`
+	Endpoint         string `json:"endpoint"`
+	Impersonate      string `json:"impersonate"`
+	SmartProxyConfig struct {
```

Comment:
probably worth to define this as a proper data type?

Reply:
Done